js/README.md

```diff
@@ -82,6 +82,36 @@ await schematic.checkFlag("some-flag-key");
 schematic.cleanup();
 ```
 
+## Troubleshooting
+
+For debugging and development, Schematic supports two special modes:
+
+### Debug Mode
+
+Enables console logging of all Schematic operations:
+
+```typescript
+// Enable at initialization
+const schematic = new Schematic("your-api-key", { debug: true });
+
+// Or via URL parameter
+// https://yoursite.com/?schematic_debug=true
+```
+
+### Offline Mode
+
+Prevents network requests and returns fallback values for all flag checks:
+
+```typescript
+// Enable at initialization
+const schematic = new Schematic("your-api-key", { offline: true });
+
+// Or via URL parameter
+// https://yoursite.com/?schematic_offline=true
+```
+
+Offline mode automatically enables debug mode to help with troubleshooting.
```

Comment:
Do we want this to be true? feels like these have different use cases. Although we can always roll this back later.

Reply:
Yeah, I mostly just did this because the offline and debug behavior for events are basically the same, had a similar thought though